src/main/java/com/minecolonies/rankup/util/Constants.java

```diff
@@ -0,0 +1,59 @@
+package com.minecolonies.rankup.util;
+
+/**
+ * Constants for our plugin.
+ */
+public final class Constants
+{
+
+    private Constants()
+    {
+        //Empty private constructor.
+    }
+
+    public static class PlayerInfo
+    {
+        private PlayerInfo()
+        {
+            //Empty private constructor.
+        }
+
+        public static final String PLAYER_NAME   = "{player}";
+        public static final String PLAYER_GROUP  = "{group}";
+        public static final String PLAYER_RANK   = "{rank}";
+        public static final String PLAYER_PREFIX = "{prefix}";
+        public static final String PLAYER_JOIN   = "{joindate}";
+        public static final String PLAYER_LAST   = "{lastjoin}";
+        public static final String PLAYER_TRACK  = "{track}";
+    }
+
+    public static class ModuleInfo
+    {
+        private ModuleInfo()
+        {
+            //Empty private constructor.
+        }
+
+        public static final String TIMING_TIME     = "{timing-time}";
+        public static final String TIMING_NEXT     = "{timing-next}";
+        public static final String ECONOMY_BAL     = "{economy-bal}";
+        public static final String ECONOMY_NEXT    = "{economy-next}";
+        public static final String MAGIBRIDGE_CURR = "{magibridge-currant}";
+        public static final String MAGIBRIDGE_NEXT = "{magibridge-next}";
+    }
+
+    public static class SQL
+    {
+        private SQL()
+        {
+            //Empty private constructor.
+        }
+
+        public static final String WHERE  = "WHERE";
+        public static final String FROM   = "FROM";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1213)